optimade/server/mappers/entries.py

```diff
@@ -44,3 +42,31 @@ def map_back(cls, doc: dict) -> dict:
         :return: A resource object in OPTiMaDe format
         :rtype: dict
         """
+        if "_id" in doc:
+            del doc["_id"]
+
+        mapping = ((real, alias) for alias, real in cls.all_aliases())
+        newdoc = {}
+        reals = {real for alias, real in cls.all_aliases()}
+        for k in doc:
+            if k not in reals:
+                newdoc[k] = doc[k]
+        for real, alias in mapping:
+            if real in doc:
+                newdoc[alias] = doc[real]
+
+        if "attributes" in newdoc:
+            raise Exception("Will overwrite doc field!")
+        attributes = newdoc.copy()
+
+        top_level_entry_fields = {"id", "type", "relationships", "links"}
```

Comment:
I like the new generic mapper, could we move this set into the class attributes (potentially to be overloaded by child mappers)?

Reply:
Will do.